src/System.Windows.Forms/src/System/Windows/Forms/ToolStripItem.cs

```diff
@@ -3199,6 +3204,11 @@ internal void RescaleConstantsInternal(int newDpi)
 
         public void Select()
         {
+            Select(forceRaiseA11yFocusChanged: false);
+        }
+
+        internal void Select(bool forceRaiseA11yFocusChanged)
```

Comment:
There are no other usages of `a11y` abbreviation in the codebase, so I would spell it out:
```suggestion
        internal void Select(bool forceRaiseAccessibilityFocusChanged)
```

Reply:
Renamed to `forceRaiseAccessibilityFocusChanged` as less confusing